### PR TITLE
LPS-180899 | FDSViews#CanAssertPagination

### PR DESCRIPTION
**Pull request by alexmds2004**
**Description**

- **Given** access to the Datasets admin page 

- **When** The user goes to add a new Dataset

- **And** Add a 6 views

- **And** Change the pagination to 4 items

- **Then** Confirm that 2 pages are shown

- **When** the user deletes two Views

- **Then** Confirm that only one page is displayed

**JIRA Ticket**
https://issues.liferay.com/browse/LPS-180899

### CODE REVIEW

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:frontend

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:frontende

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:frontend

**Comment by alexmds2004** [comment]
@mateusralv would you like to see this pr?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSViews.testcase [hunk 1]
```diff
@@ -29,6 +29,77 @@ definition {
...
+			FrontendDataSetAdmin.changePagination(itemsPerPage = "4 Items");
+		}
+
+		task ("Then Confirm that 2 pages are shown") {
+			AssertElementPresent(locator1 = "Pagination#NEXT_LINK");
+
+			for (var num : list "1,2,3,4") {
+				AssertElementPresent(
+					key_itemName = "Test Dataset ${num}",
+					locator1 = "FrontendDataSetAdmin#VIEW_PAGINATION");
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSViews.testcase:78]:
```suggestion
					entryName = "View Test ${num}",
					locator1 = "FrontendDataSetAdmin#DATA_SET_VIEW_ENTRY_NAME");
```

Comment (mateusralv) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSViews.testcase:78]: the views must have different names, the ones you created have the same name and you can use DATA_SET_VIEW_ENTRY_NAME path here and delete VIEW_PAGINATION path

**Review by mateusralv — CHANGES_REQUESTED** [review]
Hey @alexmds2004 could you take a look? Thanks 😄

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:frontend

**Comment by alexmds2004** [comment]
ci:test:frontend

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:frontend

**Comment by alexmds2004** [comment]
@mateusralv

**Review by mateusralv — APPROVED** [review]

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
ci:help

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
continue: https://github.com/liferay-frontend/liferay-portal/pull/3205

**Pull request closed without merging**